src/hooks/useVerifiableCredential.test.ts

```diff
@@ -21,7 +21,9 @@ jest.mock('wagmi', () => ({
   useAccount: jest.fn(),
   usePublicClient: jest.fn(),
   useSignTypedData: jest.fn(),
-  useChainId: () => 1,
```

Comment:
you dont need that when u change wagmi-config.ts

Reply:
fixed